aspnet/mvc/views/overview.rst

```diff
@@ -0,0 +1,223 @@
+
+.. code-block:: html
+    :emphasize-lines: 1
+
+    @model WebApplication1.ViewModels.Address
+    <h2>Contact</h2>
+    <address>
+        @Model.Street<br />
+        @Model.City, @Model.State @Model.PostalCode<br />
+        <abbr title="Phone">P:</abbr>
+        425.555.0100
+    </address>
+
+Once a model has been specified for a view, the instance sent to the view can be accessed in a strongly-typed manner using ``@Model`` as shown above. To provide an instance of the model type to the view, the controller passes it as a parameter:
+
+.. code-block:: c#
+    :emphasize-lines: 13
+   
+    public IActionResult Contact()
+    {
+        ViewData["Message"] = "Your contact page.";
+
+        var viewModel = new Address()
+        {
+            Name = "Microsoft",
+            Street = "One Microsoft Way",
+            City = "Redmond",
+            State = "WA",
+            PostalCode = "98052-6399"
+        };
+        return View(viewModel);
+    }
+
+There are no restrictions on the types that can be provided to a view as a model. We recommend passing Plain Old CLR Object (POCO) view models with little or no behavior, so that business logic can be encapsulated elsewhere in the app. An example of this approach is the *Address* viewmodel used in the example above:
+
+.. code-block:: c#
+    :emphasize-lines: 13
+   
+    namespace WebApplication1.ViewModels
+    {
+        public class Address
+        {
+            public string Name { get; set; }
+            public string Street { get; set; }
+            public string City { get; set; }
+            public string State { get; set; }
+            public string PostalCode { get; set; }
+        }
+    }
+
+.. note:: Nothing prevents you from using the same classes as your business model types and your display model types. However, keeping them separate allows your views to vary independently from your domain or persistence model, and can offer some security benefits as well (for models that users will send to the app using :doc:`model binding </mvc/models/model-binding>`).
+
+Loosely Typed Data
+^^^^^^^^^^^^^^^^^^
+
+In addition to strongly typed views, all views have access to a loosely typed collection of data. This same collection can be referenced through either the ``ViewData`` or ``ViewBag`` properties on controllers and views. The ``ViewBag`` property is a wrapper around ``ViewData`` that provides a dynamic view over that collection. It is not a separate collection.
+
+``ViewData`` is a dictionary object accessed through ``string`` keys. You can store and retrieve objects in it, and you'll need to cast them to a specific type when you extract them. You can use ``ViewData`` to pass data from a controller to views, as well as within views (and partial views and layouts). String data can be stored and used directly, without the need for a cast.
+
+.. code-block:: html
+    :emphasize-lines: 13,16
+   
+    @{
+        ViewData["greeting"] = "Hello";
```

Comment:
I would generally agree, but all of the sample templates do it already, which is why I went with it (since I could show a single file in the code block, instead of two). Here's what I mean:
```
@{
    ViewData["Title"] = "About";
}
<h2>@ViewData["Title"].</h2>

```

If you like, I'll update so that the setting is happening in an action, though.